cli/sawtooth_cli/batch.py

```diff
@@ -56,7 +59,9 @@ def add_batch_parser(subparsers, parent_parser):
         Shows the data for a single batch, or for a particular property within
```

Comment:
I know these aren't yours, but could we remove the extra indent from the first line of these epilogs? It looks weird to me now that it is formatted properly.